### PR TITLE
[enhance](cloud)support SHOW-TRANSACTION

### PR DESCRIPTION
**Pull request by cjj2010**
## Proposed changes

Issue Number: close #xxx

now only support eg：
SHOW TRANSACTION WHERE ID=xxxxx；

SHOW TRANSACTION WHERE LABEL = 'xxxxx';

### CODE REVIEW

File: fe/fe-core/src/main/java/org/apache/doris/cloud/transaction/TxnUtil.java [hunk 1]
```diff
@@ -366,4 +377,46 @@ public static TransactionState transactionStateFromPb(TxnInfoPB txnInfo) {
         }
         return transactionState;
     }
+
+    public static List<String> getTxnStateInfo(TransactionState txnState, List<String> info) {
```
Comment (gavinchou) [inline fe/fe-core/src/main/java/org/apache/doris/cloud/transaction/TxnUtil.java:381]: keep it in DatabaseTransactionMgr

File: fe/fe-core/src/main/java/org/apache/doris/cloud/transaction/TxnUtil.java [hunk 2]
```diff
@@ -366,4 +377,46 @@ public static TransactionState transactionStateFromPb(TxnInfoPB txnInfo) {
...
+        info.add(TimeUtils.longToTimeString(txnState.getFinishTime()));
+        info.add(txnState.getReason());
+        info.add(String.valueOf(txnState.getErrorReplicas().size()));
+        info.add(String.valueOf(txnState.getCallbackId()));
+        info.add(String.valueOf(txnState.getTimeoutMs()));
+        info.add(txnState.getErrMsg());
+        return info;
+    }
+
+    public static void checkAuth(long dbId, TransactionState txnState) throws AnalysisException {
```
Comment (gavinchou) [inline fe/fe-core/src/main/java/org/apache/doris/cloud/transaction/TxnUtil.java:400]: ditto

File: fe/fe-core/src/main/java/org/apache/doris/cloud/transaction/TxnUtil.java
Comment (gavinchou) [inline fe/fe-core/src/main/java/org/apache/doris/cloud/transaction/TxnUtil.java:1]: consider move TxnUtil to apache/doris/transaction/

**Review by gavinchou — CHANGES_REQUESTED** [review]

File: fe/fe-core/src/main/java/org/apache/doris/cloud/transaction/TxnUtil.java
Comment (cjj2010) [reply fe/fe-core/src/main/java/org/apache/doris/cloud/transaction/TxnUtil.java:1]: TxUtil includes cloud domain functionality. I have recreated TransactionUtil under Apache/toris/transaction/for use by all domains. Would you like to see if it is suitable

**Review by gavinchou — APPROVED** [review]
LGTM

**Comment by dataroaring** [comment]
run buildall

**Comment by cjj2010** [comment]
run buildall

**Comment by cjj2010** [comment]
run buildall

**Comment by cjj2010** [comment]
run buildall

**Review by dataroaring — APPROVED** [review]
LGTM

**Pull request merged**